swap_meet/clothing.py

```diff
@@ -1,2 +1,12 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, id = None, condition = 0, fabric = "Unknown"):
+        super().__init__(id, condition)
+        self.fabric = fabric
+
+    def get_category(self):
+        return "Clothing"
+    
+    def __str__(self):
+        return (f"{super().__str__()} It is made from {self.fabric} fabric.")
```

Comment:
👍 Good job not repeating string values by calling the `super str` dunder method!